ansible/roles/openstack-idr-instance/tasks/floatingip.yml

```diff
@@ -0,0 +1,30 @@
+---
+# Handle floating IPs with multiple NICs
+# The os_floating_ip module may behave in an unexpected manner.
+# These tasks attempt to work around them.
+# Even so you may still see problem with Ansible 2.1, if so try 2.2 or 2.3
+
+- name: idr vm | floating ip
```

Comment:
Now blocked on testing the idr-0-3-1 branch due to this. Also with ansible 2.2.0.0